lib/l10n/WalletLocalizations.dart

```diff
@@ -0,0 +1,201 @@
+      'backup_words_order_content': '请按顺序点击助记词，以确认您\n正确备份。',
+      'backup_words_order_finish': '完成',
+
+      'welcomePageOneTitle' : '欢迎加入 Omni 平台！',
+      'welcomePageOneContent' : "为了您的安全，请您抽时间来了解一些重要信息。\n\n"
+          "如果您访问了钓鱼网站或丢失备份短语（SEED短语），我们无法恢复您的资金或冻结您的帐户。\n\n"
+          "如您愿意继续使用我们的平台，您同意接受与您SEED损失相关的所有风险。如果您丢失了SEED，"
+          "您同意并承认 Omni 平台不会对此造成的负面后果承担责任。",
+      'welcomePageOneButton' : '您需要知道的关于您的SEED信息',
+      
+      'welcomePageTwoTitle' : '您需要知道关于您的SEED的信息',
+      'welcomePageTwoContentOne' : "注册您的账户时，您要保存您的密码（SEED）并使用密码来保护您的账户。"
+          "用普通的中央服务器上，您特别注意密码, 而且您可以通过电子邮件更改和重置密码。然而，Omni 与众不同 — "
+          "该钱包安全地储存在您使用的设备上 ：",
+      'welcomePageTwoContentTwo' : "您匿名地使用您的钱包，指的是您的账户未连接到电子邮件账户或任何其他识别数据。",
+      'welcomePageTwoContentThree' : "用某个设备或浏览器时，您的密码可以保护您的账户，"
+          "为了确保您的密码短语不被保存在存储器中。",
+      'welcomePageTwoContentFour' : "如果您忘记了密码，您可以通过您的密码短语使用账户恢复表单并轻松创建"
+          "一个新密码。但是，如果您遗失了密码，您将无法访问您的帐户。",
+      'welcomePageTwoContentFive' : "你不能改变你的秘密短语。如果您不小心将其发送给某人或怀疑诈骗者已将"
+          "其移交给他人，请立即创建一个新的 Omni 钱包并将资金转入其中。",
+      'welcomePageTwoButtonBack' : '    返回    ',
+      'welcomePageTwoButtonNext' : '    保护自己    ',
+
+      'welcomePageThreeTitle' : '如何防止网络钓鱼攻击',
+      'welcomePageThreeContentOne' : "诈骗最常见的攻击方式之一是网络钓鱼，一般钓鱼者在Facebook或其他类"
+          "似于真实的网站上创建假社群。",
+      'welcomePageThreeContentTwo' : "请总是检查URL: https://www.lunarx_omni.com",
+      'welcomePageThreeContentThree' : "访问您的账户时请不要使用具有扩展程序或插件的浏览器。",
+      'welcomePageThreeContentFour' : "请不要打开来自未知发件人的电子邮件或链接。",
+      'welcomePageThreeContentFive' : "请您定期更新您的浏览器和操作系统。",
+      'welcomePageThreeContentSix' : "请您使用官方安全软件。请不要安装会被黑客攻击的未知软件。",
+      'welcomePageThreeContentSeven' : "使用公共Wi-Fi或其他人的设备时请勿访问您的钱包。",
+      'welcomePageThreeButtonBack' : '    返回    ',
+      'welcomePageThreeButtonNext' : '    我了解    ',
+
+      'startPageAppBarTitle' : 'Omni 钱包',
+      'startPageButtonFirst' : '      开始使用      ',
+      'startPageButtonSecond' : '      恢复钱包      ',
+      'startPageLanguageBarTitle' : '语言',
+
+      'main_page_title' : '钱包',
+      'common_btn_skip' : '跳过',
+    },
+
+    'en': {
+      'main_index_title': 'LunarX_Omni Wallet',
+      'backup_index_prompt_btn': 'I got it',
+      'backup_index_title': 'Backup the wallet',
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
+      'backup_index_prompt_tips': 'Anyone who gets your mnemonic will get your assets. Please copy on the paper for safekeeping.',
+      'backup_index_prompt_title': "Don't screenshots",
+      'backup_words_title': 'Back up your mnemonic',
+      'backup_words_next': 'next',
+      'backup_words_content': "Please copy the mnemonic word carefully,\nwe will verify it in the next step.",
```

Comment:
It looks like you might be displaying more than 1 word at a time, if this is the case then this should be pluralized. 
`word` -> `words`
`verify it` -> `verify them`

Reply:
Fixed.